pkg/appimport/testing/db-compose.yaml

```diff
@@ -1,21 +0,0 @@
-version: '2'
-
-services:
-  local-test-db:
-    container_name: local-test-db
-    image: $DDEV_DBIMAGE
-    volumes:
-      - "./data:/db"
-    restart: always
-    environment:
-      MYSQL_DATABASE: data
-      MYSQL_ROOT_PASSWORD: root
-    ports:
-      - "3306"
-    labels:
-      com.ddev.site-name: test
-      com.ddev.container-type: db
-networks:
-  default:
-    external:
-      name: ddev_default
```

Comment:
Correct, this is a leftover from functionality and tests that were recently removed.